test/run.sh

```diff
@@ -1,15 +1,46 @@
-#!/bin/bash -eu
```

Comment:
I got rid of `-u` because it was complaining locally that `$CIRCLECI` wasn't set.

Reply:
Ah perfect, I thought there must be something like that. I'll make that change.